instrumentation/grpc-1.6/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/grpc/v1_6/GrpcSingletons.java

```diff
@@ -30,6 +31,7 @@ public final class GrpcSingletons {
     GrpcTelemetry telemetry =
         GrpcTelemetry.builder(GlobalOpenTelemetry.get())
             .setCaptureExperimentalSpanAttributes(experimentalSpanAttributes)
+            .setRequestMetadataValuesToCapture(CommonConfig.get().getRpcRequestMetadata())
```

Comment:
The grpc metadata config isn't really "common" - it's supposed to be used only in this instrumentation. I think it should be fine to just inline it and remove it from `CommonConfig`

Reply:
Changing it